### PR TITLE
[WIP] Docs: fixed misleading unnamespaced event dispatch

### PR DESCRIPTION
**Pull request by klimesf**

### CODE REVIEW

File: docs/en/index.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ class FooListener extends Nette\Object implements Kdyby\Events\Subscriber
 Now when I invoke the event `onStartup`, the listener method should get called
 
 ```php
+namespace Nette\Application;
```
Comment (enumag) [inline docs/en/index.md]: Nope, don't use Nette namespace.

Comment (klimesf) [inline docs/en/index.md]: It's also used later. Maybe we can come up with better examples.

Comment (enumag) [inline docs/en/index.md]: Oh I see. But this doesn't really have anything to do with Nette. Remove the namespace and use this:
`$this->evm->dispatchEvent('Application::onStartup', new EventArgsList(array($this)));`

**Comment by klimesf** [comment]
Please leave this open. This is not perfect, but the original version was confusing (at least for me). I will push some more changes next week, after I return from vacation. Opinions welcome.

**Pull request closed without merging**